package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "todo_list",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "dev": "webpack --mode development --watch",
+    "build": "webpack --mode production",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
- [ ] After installing your dependencies by running `npm install`, Then I try running `npm run start` but I got the error below. I checked your `package.json` file and found out that the `start` script which is responsible for starting your app in the browser is missing, I added the `start` script `"start": "webpack serve --open"` and I was able to run your app with `npm run start`. Kindly add the start script in the package.json.

<img width="390" alt="Code_LyeIp0i7vr" src="https://user-images.githubusercontent.com/81468675/156262884-b31d30c2-b27a-47b1-998e-ca0ce4150a3b.png">